tango/__main__.py

```diff
@@ -267,12 +267,10 @@ def cleanup(*args, **kwargs):
     "-j",
     "--parallelism",
     type=int,
-    help="""The maximum number of steps to run in parallel (if possible).
-    By default there is no parallelism, but each step is ran in its own subprocess (same as setting '-j=1').
-    A value of 0 or less means each step is ran in the main process using the default executor.
-    A value greater than 1 means at most 'j' steps will be ran in parallel,
-    each within its own subprocess, using the multicore executor.
-    This option is ignored if you've configured your executor in a 'tango.yml' settings file.""",
+    help="""The maximum number of steps to run in parallel (for executors that support this).
+    The exact behavior depends on the executor. If you're using the default executors,
+    a value of 0 (or left unspecified) means each step is ran in the main process using the default executor,
```

Comment:
```suggestion
    a value of 0 (or left unspecified) means each step is run in the main process using the default executor,
```

Reply:
Ugh lol this is 2nd time you've caught me with this typo 🤦‍♂️